### ISSUE TITLE
add inspector for appium desktop

### ISSUE THREAD

**Issue by jlipps** [appium/appium#7539]
Acceptance criteria:
* [ ] screenshot displayed and automatically updated on actions
* [ ] navigable view hierarchy of elements
* [ ] clicking on an element in hierarchy shows element details (access. id, etc)
* [ ] clicking on an element highlights element bounds in screenshot
* [ ] palette of simple actions (tap, send keys) active when element is selected
* [ ] highlight element bounds and show in hierarchy based on x,y of click/hover on screenshot

**Comment by mykola-mokhnach** [comment]
how about element search helper (+ different strategies support)?

**Comment by jlipps** [comment]
@mykola-mokhnach I don't quite get what you have in mind, can you share more?

**Comment by mykola-mokhnach** [comment]
sure: it should be possible to enter let's say xpath, accessibility id or class name and the inspector will highlight all found matches

**Comment by jlipps** [comment]
I think that's a great feature idea. I don't want to make it part of the acceptance criteria for this issue though, since I think we can provide a lot of value without that feature as well, and it'd be nice to start shipping an inspector.

**Comment by mykola-mokhnach** [comment]
yup. It's partially implemented in that [inspector](https://github.com/mykola-mokhnach/Appium-iOS-Inspector) (only xpath search helper is implemented), which is maintained by myself, and my colleagues find this feature very useful.
